topics/statistics/tutorials/clustering-machinelearning/tutorial.md

```diff
@@ -0,0 +1,482 @@
+---
+layout: tutorial_hands_on
+
+title: 'Clustering in Machine Learning'
+zenodo_link: https://zenodo.org/...
+questions:
+- How to use clustering methods to categorized data in different clusters
+objectives:
+- Learn clustering background
+- Learn hierarchical clustering algorithm
+- Learn k-means clustering algorithm
+- Learn DBSCAN clustering algorithm
+- Apply clustering algorithms to different data sets
+- Learn how to visualize the results
+key_points:
+- Using clustering methods, category and cluster of the data are learned using hirarchical, k-means and DBSCAN
+- For each clustering algorithm, number of clusters and the algorithm parameters should be optimised based on the dataset
+time_estimation: 2H
+contributors:
+- khanteymoori
+- anuprulez
+---
+
+# Introduction
+{:.no_toc}
+
+In this tutorial we will discuss an unsupervised learning task called clustering. The goal of unsupervised learning is to discover hidden structure or patterns in unlabeled training data.  Clustering is used to find groups of similar observations within a set of unlabeled data, such that members of the same group, or cluster, are more similar to each other by a given metric than they are to the members of the other clusters. Clustering is commonly used to explore a dataset. For example, in biology, clustering is used to find groups of genes with similar expression patterns. We cluster objects in multiple groups , such that objects within the same cluster are as similar as possible (i.e., high intra-class similarity), whereas objects from dierent clusters are as dissimilar as possible (i.e., low inter-class similarity).
```

Comment:
`Clustering is commonly used to explore a dataset`. How?

Reply:
First of all, what is a cluster and why is it important to find clusters in data?

Reply:
`objects in multiple groups , such`. Extra space before a comma.

Reply:
`dierent`. A typo

https://app.grammarly.com/ has a Chrome extension to check typos and grammar. Its a really useful tool :)

Reply:
The introduction is modified and the comments are applied.